File: WinThant16-cs50-problems-2020-x-mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int number;
    do
    {
        number = get_int("Height: ");
    }
    while (number < 1 || number > 8);

    //asking for pyramid height
    for (int i = 0; i < number; i++)
    {
        //space
        for (int j = number - 1; j > i; j--)
        {   
            printf(" ");
        }    
        //hash
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: WinThant16-cs50-problems-2020-x-sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    height = get_pos_int()
    i = 0
    j = 0
    k = 0
    for i in range(height):

        for j in range(height-i-1):
            print(" ", end="")
        for k in range(i+1):
            print("#", end="")
        
        print("  ", end="")
        
        for l in range(i+1):
            print("#", end="")
        print()
        

# verifying height between 0 and eight
def get_pos_int():
    # asking pyramid height
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break
    return height


main()